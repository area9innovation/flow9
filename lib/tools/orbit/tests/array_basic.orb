// Basic test for array syntax

// Single element array
[1];

// Multiple element array
[2, 3, 5];

// Empty array
[];

// Nested arrays
[[1, 2], [3, 4]];