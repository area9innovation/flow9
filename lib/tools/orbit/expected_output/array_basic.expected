Flow compiler (3rd generation)

Processing 'orbit' on http server

Result: [[1, 2], [3, 4]]
VERIFY - THIS OUTPUT NEEDS HUMAN VERIFICATION
